components/drafts/drafts_link/drafts_tour_tip/drafts_tour_tip.tsx

```diff
@@ -0,0 +1,119 @@
+
+import Badge from 'components/widgets/badges/badge';
+
+import {Preferences} from 'mattermost-redux/constants';
+
+import {showDraftsPulsatingDotAndTourTip} from 'selectors/drafts';
+import {TourTip, useMeasurePunchouts} from '@mattermost/components';
+import {setDraftsTourTipPreference} from 'actions/views/drafts';
+
+const title = (
+    <span className='title-with-new-badge'>
+        <FormattedMessage
+            id='drafts.tutorialTip.title'
+            defaultMessage='Drafts'
+        />
+        <Badge variant='success'>
+            <FormattedMessage
+                id='badge.label.new'
+                defaultMessage='NEW'
+            />
+        </Badge>
+    </span>
+);
+
+const screen = (
+    <>
+        <FormattedMessage
+            id='drafts.tutorialTip.description'
+            defaultMessage='With the new Drafts view, all of your unfinished messages are collected in one place. Return here to read, edit, or send draft messages.'
+        />
+    </>
+
+);
+
+const prevBtn = (
+    <FormattedMessage
+        id='drafts.tutorial_tip.notNow'
+        defaultMessage='Not now'
+    />
+);
+
+const nextBtn = (
+    <FormattedMessage
+        id='drafts.tutorial_tip.viewDrafts'
+        defaultMessage='View drafts'
+    />
+);
+
+const DraftsTourTip = () => {
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    const showTip = useSelector(showDraftsPulsatingDotAndTourTip);
+    const {url} = useRouteMatch();
+    const nextUrl = `${url}/drafts`;
+
+    const [tipOpened, setTipOpened] = useState(showTip);
+
+    const handleDismiss = useCallback((e: React.MouseEvent) => {
+        e.stopPropagation();
+        dispatch(setDraftsTourTipPreference({[Preferences.DRAFTS_TOUR_TIP_SHOWED]: true}));
+        setTipOpened(false);
+    }, []);
+
+    const handleNext = useCallback(() => {
+        dispatch(setDraftsTourTipPreference({[Preferences.DRAFTS_TOUR_TIP_SHOWED]: true}));
+        setTipOpened(false);
+        history.push(nextUrl);
+    }, []);
+
+    const handleOpen = useCallback((e: React.MouseEvent) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        if (tipOpened) {
+            dispatch(setDraftsTourTipPreference({[Preferences.DRAFTS_TOUR_TIP_SHOWED]: true}));
+            setTipOpened(false);
+        } else {
+            setTipOpened(true);
+        }
+    }, []);
+
+    const overlayPunchOut = useMeasurePunchouts(['sidebar-drafts-button'], []);
+
+    return (
+        <>
+            {
+                (showTip) &&
+                <TourTip
+                    show={tipOpened}
+                    screen={screen}
+                    title={title}
+                    overlayPunchOut={overlayPunchOut}
+                    placement='right-start'
+                    pulsatingDotPlacement='right'
+                    step={1}
+                    singleTip={true}
+                    showOptOut={false}
+                    interactivePunchOut={true}
```

Comment:
I think interactive isn't working as expected, so better would be to make `interactivePunchOut={false}`. A contributor is working on it. Otherwise, I will fix it soon.

Reply:
I've set it to false for now 👍